make/autoconf/flags-cflags.m4

```diff
@@ -636,8 +636,7 @@ AC_DEFUN([FLAGS_SETUP_CFLAGS_HELPER],
   # JDK libraries.
   STATIC_LIBS_CFLAGS="-DSTATIC_BUILD=1"
   if test "x$TOOLCHAIN_TYPE" = xgcc || test "x$TOOLCHAIN_TYPE" = xclang; then
-    STATIC_LIBS_CFLAGS="$STATIC_LIBS_CFLAGS -ffunction-sections -fdata-sections \
-      -DJNIEXPORT='__attribute__((visibility(\"hidden\")))'"
+    STATIC_LIBS_CFLAGS="$STATIC_LIBS_CFLAGS -ffunction-sections -fdata-sections"
```

Comment:
Is it better/clearer to make the use of the default explicit as we do in the source:
```
#define JNIEXPORT     __attribute__((visibility("default")))
```
?

Reply:
@dholmes-ora Do you mean something like this (instead of removing the `-D` expression)?

```
STATIC_LIBS_CFLAGS="$STATIC_LIBS_CFLAGS -ffunction-sections -fdata-sections \
      -DJNIEXPORT='__attribute__((visibility(\"default\")))'"
```

I can surely do that if that's preferred.

Reply:
I've done that now. Let me know if that's what you had in mind.